app.rb

```diff
@@ -0,0 +1,151 @@
+require_relative 'classroom'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
+require_relative 'person'
+
+class App
+  def initialize
+    @my_books = []
+    @my_rentals = []
+    @people = []
+  end
+
+  def my_menu
+    puts "\n\nPlease choose an option by entering a number:"
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
+
+  def run
+    choice = my_menu
+    case choice
+    when '1'
+      action_list_books
+    when '2'
+      action_list_people
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rental
+    else
+      puts 'Thanks for using this app ...'
+      exit
+    end
+  end
+
+  def list_books
+    @my_books.each_with_index do |x, index|
+      puts "#{index}) Title: \"#{x.book}\", Author: #{x.author}\n"
+    end
+  end
+
+  def list_people
+    @people.each_with_index do |x, index|
+      puts "#{index}) [#{x.class.name}] Name: #{x.name}, ID: #{x.id}, Age: #{x.age}\n"
+    end
+  end
```

Comment:
When listing people student get render correctly but for teacher the name and age field render wrong values. Kindly, fix that.

<img width="1292" alt="image" src="https://user-images.githubusercontent.com/34578631/180601261-d021726d-001b-4ca6-b9f8-8468d0d81cd0.png">